packages/jest-circus/src/__tests__/circusItTestError.test.ts

```diff
@@ -35,21 +30,18 @@ describe('test/it error throwing', () => {
   });
   it(`it throws error with missing callback function`, () => {
     expect(() => {
-      // $FlowFixMe: Easy, we're testing runtime errors here
```

Comment:
you don't need `@ts-ignore`? Odd

Reply:
Currently I need type for `it` and `test` APIs then I can see errors. Now all of them are giving error